public/index.html

```diff
@@ -9,7 +9,8 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="src\images\icons8-apple-music-48.png" />
+    <!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
```

Comment:
not at all, deleted👌

Reply:
@akosasante just a small doubt about the difference between these two tags, since the first one is a common favicon and the second one is for iOS devices- so the first should have an .ico extention to be served correctly in most browsers and the second  should be  .png , because iOS supports more this format, right? 
![image](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/103755090/b56d8569-97d0-436b-8d49-2e4044785917)